plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/project/GcsDataflowProjectClient.java

```diff
@@ -88,13 +89,16 @@ public StagingLocationVerificationResult createStagingLocation(
     }
   }
 
-  private static String toGcsBucketName(String stagingLocation) {
+  public static String toGcsBucketName(String stagingLocation) {
```

Comment:
I'll just not process leading or trailing whitespace. See https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/2192#discussion_r129118057.